source/_components/netdata.markdown

```diff
@@ -43,23 +43,71 @@ $ curl -X GET "http://[Netdata_Instance]:19999/api/v1/data?chart=[data_group]&po
 - `dimension_names`: Names shown in the frontend.
 - `dimension_ids`: Names to use for `element`.
 
+Alternatively you can browse to the built in Netdata API in your browser `http://[Netdata_Instance]:19999/api/v1/allmetrics?format=json` and search for the `data_group` identified in the Netdata frontend. In the example JSON below the data group is "system.load".
+
+```json
+	"system.load": {
+		"name":"system.load",
+		"context":"system.load",
+		"units":"load",
+		"last_updated": 1558446920,
+		"dimensions": {
+			"load1": {
+				"name": "load1",
+				"value": 0.1250000
+			},
+			"load5": {
+				"name": "load5",
+				"value": 0.1290000
+			},
+			"load15": {
+				"name": "load15",
+				"value": 0.1430000
+			}
+		}
+	},
+```
+
+Once the `data_group` "system.load" and the `element` "load15" have been identified from the JSON it can be configured in your configuration.yaml like the example below. 
+
 ## {% linkable_title Configuration %}
 
-To add this platform to your installation, add the following to your `configuration.yaml` file:
+To add this component to your installation, add the following to your `configuration.yaml`.
```

Comment:
Per definition is this integration a sensor platform. Also, it's a configuration file.

Reply:
Changed wording for adding to the configuration.yaml